src/cmd-buildextend-metal

```diff
@@ -63,17 +63,8 @@ if [ $# -ne 0 ]; then
 fi
 
 case "$basearch" in
-    "x86_64"|"aarch64"|"s390x"|"ppc64le") use_anaconda=;;
-    *)
-        # for qemu, we can fallback to Anaconda
-        if [[ ${image_type} == qemu ]]; then
-            use_anaconda=1
-        else
-            # otherwise, we don't know how to create bare metal images for this
-            # architecture
-            fatal "$basearch is not supported for this command"
-        fi
-        ;;
+    "x86_64"|"aarch64"|"s390x"|"ppc64le") ;;
+    *) fatal "$basearch is not supported for this command" ;;
```

Comment:
Hmm, would it make sense to just move it somewhere near the top of `cmdlib.sh` and have a more generic error?

Reply:
Possibly - though I'd like to keep this PR scoped to mostly just deleting code so we can back it out more easily if need be.

However, thinking about this one more - it feels a bit harsh to people porting to other architectures if we make cosa refuse to work at all, versus just the disk image building part.  So I think this probably makes sense here still.